staff.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="./assets/css/staff.css">
+
+        <title>Server Staff</title>
+    </head>
+    <body>
+        <h1>Codecademy Community Server Staff:</h1>
+
+        <h2>Admins</h2>
+        <div class="staff-list">
+            <div class="staff-card">
+                <img class="profile-picture" src="https://cdn.discordapp.com/avatars/608641641284370462/2d2d9cb6e846fceb586e3630186f3508.png?size=128">
+                <p class="name super-admin">Brussels</p>
+                <p class="username">brussels#0001</p>
+            </div>
+            <div class="staff-card">
+                <img class="profile-picture" src="https://cdn.discordapp.com/avatars/106136650161061888/bcd227d0034ea445f5773e800f8a5880.png?size=128">
+                <p class="name super-admin">Fede [Codecademy HQ]</p>
+                <p class="username">Fede#3591</p>
+            </div>
+            <div class="staff-card">
+                <img class="profile-picture" src="https://cdn.discordapp.com/avatars/711756182909485116/ce00fb04adfd36519d16627f1dffea3d.png?size=80">
+                <p class="name admin">el_cocodrilo</p>
+                <p class="username">el_cocodrilo#7140</p>
+            </div>
+            <div class="staff-card">
+                <img class="profile-picture" src="https://cdn.discordapp.com/avatars/610527108254400559/19ed8d3eaa42649a35f8e8cf0dc9cd3d.webp?size=80">
+                <p class="name admin">Lillian (she/her)</p>
+                <p class="username">Lillian [Codecademy HQ]#8166</p>
+            </div>
+            <div class="staff-card">
+                <img class="profile-picture" src="assets/img/discord-blank.png">
+                <p class="name admin">mariana-hq</p>
+                <p class="username">mariana-hq#2524</p>
+            </div>
+        </div>
+
+        <h2>Moderators</h2>
+        <div class="staff-list">
+            <div class="staff-card">
+                <img class="profile-picture" src="https://cdn.discordapp.com/avatars/711997187290300546/8e0fd8d7509fe2876f276f1ac10f60bf.webp?size=128">
+                <p class="name mod">8-Bit</p>
+                <p class="username">Shaylin#5862</p>
+            </div>
+            <div class="staff-card">
+                <img class="profile-picture" src="https://cdn.discordapp.com/avatars/655902608526934017/a_b83ade74fbeb14d88ac8f1072bf40ba3.gif?size=80">
+                <p class="name mod">Calla</p>
+                <p class="username">squidm0#3780</p>
+            </div>
+            <div class="staff-card">
+                <img class="profile-picture" src="https://cdn.discordapp.com/avatars/797235974807224391/1f51508c6040703eec08dd5bfda83ad1.webp?size=128">
```

Comment:
Unfortunately, Calla recently stepped back from the role. Can you remove this section?